src/modules/schedule/typedefs/schedule.ts

```diff
@@ -68,11 +75,11 @@ const typedef = gql`
     """
     Takes in schedule fields, creates a new schedule record in the database, and returns the schedule.
     """
-    createNewSchedule(created_by: String!, term: String!, schedule_name: String, is_public: Boolean, class_IDs: [String!], primary_section_IDs: [String!], secondary_section_IDs: [String!]): Schedule
+    createNewSchedule(created_by: String!, term: TermInput!, schedule_name: String, is_public: Boolean, class_IDs: [String!], primary_section_IDs: [String!], secondary_section_IDs: [String!]): Schedule
```

Comment:
As Kev suggested, consider defining an [`input` type](https://graphql.org/graphql-js/mutations-and-input-types/) for arguments for cleanliness.